src/transactions/simulation/TxSimUtils.cpp

```diff
@@ -176,6 +220,22 @@ generateScaledLiveEntries(std::vector<LedgerEntry>& entries,
             replaceIssuer(newEntry.data.offer().buying, partition);
             replaceIssuer(newEntry.data.offer().selling, partition);
             break;
+        case CLAIMABLE_BALANCE:
+        {
+            replaceIssuer(newEntry.data.claimableBalance().asset, partition);
+            mutateScaledAccountID(newEntry.data.claimableBalance().createdBy,
```

Comment:
This needs to be rebased, `createdBy` no longer exists.

Reply:
Rebased and fixed